css/main.css

```diff
@@ -262,3 +262,447 @@ textarea {
   }
 }
 
+
+/*AGATA*/
+/*MENU*/
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  /*background-color: black;*/
+}
+
+nav {
+  text-align: right;
+  background-image: linear-gradient(#4574e9, #7fa1ec);
+}
+
+
+nav > a, i {
+  font-size: 14px;
+  color: rgba(253, 253, 254, 0.89);
+  transition: .2s;
+}
+
+nav > a {
+  display: inline-block;
+  padding: 18px;
+  font-family: "Calibri", sans-serif;
+  letter-spacing: 1px;
+}
+
+a > img {
+  margin-right: 30vw;
+  width: 100px;
+
+}
+
+nav a:hover, i:hover {
+  color: #51fd44;
+}
+
+nav > a > i {
+  padding: 10px;
+  margin: auto;
+}
+
+div.black {
+  background-color: black;
+  width: 100%;
+  height: 5vh;
+}
+
+/*Header*/
+
+section.welcome {
+  position: relative;
+  margin-top: -22px;
+  padding-right: 3vw;
+  padding-left: 3vw;
+  color: #fdfdfe;
+  height: 60vh;
+  background-image: url(../img/banner.jpg);
+}
+
+img.car {
+  width: 40%;
+  position: absolute;
+  right: 2vw;
+  top: 5vh;
+}
+
+h1, h2, p, img.car, button {
+  font-family: sans-serif;
+}
+
+h1, h2 {
+  text-transform: uppercase;
+  letter-spacing: 1px;
+}
+h1{
+  color: #51fd44;
+  padding-top: 20px;
+}
+section.welcome > p {
+  font-size: 12px;
+  margin-top: 10px;
+  max-width: 60%;
+}
+button {
+  background-color: rgba(253, 253, 254, 0.89);
+  font-family: sans-serif;
+  font-size: 12px;
+  padding: 10px 15px;
+  margin: 20px 5px;
+  border: none;
+  text-transform: uppercase;
+}
+
+
+/*Sekcja Features*/
+section.features {
+  display: flex;
+  justify-content: space-around;
+  max-width: 800px;
+  margin: 0 auto;
+}
+
+
+.features h4 {
+  color: black;
+  font-weight: normal;
+  text-transform: uppercase;
+  padding: 20px 5px 5px;
+  font-family: "Roboto Light", sans-serif;
+}
+
+div.feature {
+  z-index: 999;
+  height: 170px;
+  background-color: white;
+  color: darkgrey;
+  float: left;
+  font-size: 14px;
+  text-align: center;
+  margin: -40px 20px;
+  overflow: hidden;
+  box-shadow: 0px 0px 15px grey;
+  padding: 5px -10px;
+}
+
+.clearfix::after {
+  content: "";
+  display: block;
+}
+div.feature > img {
+  display: flex;
+}
+
+button.arrow {
+  position: absolute;
+  right: 16%;
+  bottom: 22%;
+  border-radius: 45px;
+  height: 35px;
+  width: 35px;
+}
+
+button.arrow > p {
+  line-height: 17px;
+  margin: 0 3px 0 0 ;
+  font-weight: bold;
+  font-size: 14px;
+}
+
+/*Main*/
+
+main {
+  padding: 100px 0;
+}
+
+main > section > h2 {
+  color: black;
+  text-transform: none;
+  text-align: center;
+  font-size: 20px;
+  font-family: sans-serif;
+  text-decoration-line: underline;
+  text-decoration-color: #4574e9;
+  margin: 10px 0;
+}
+
+main > section > p {
+  background: white;
+  color: black;
+  text-align: center;
+  font-family: sans-serif;
+  font-size: 14px;
+}
+
+/*Main - section about*/
+
+section.about > div {
+  position: absolute;
```

Comment:
Ustawiając coś takiego rozwaliłaś cały box model. Parent przez to, że dałaś position absolute na wszystkie divy wewnątrz nie posiada w sumie, żadnego elementu. Zmieniając cokolwiek widok wygląda tak jak poniżej.

![image](https://user-images.githubusercontent.com/20086870/202054904-0b6a8116-7fd2-43b8-af8a-58902bbfe66c.png)